docs/api.rst

```diff
@@ -4770,13 +4775,13 @@ Getting Balancer events
 
        - tx_hash: The transaction hash of the event.
        - log_index: The index of the event in the transaction.
-       - timestamp: The Unix epoch point in UTC time when the event happened (in seconds).
+       - timestamp: The Unix timestamp in UTC when the event happened (in seconds).
        - event_type: The type of interaction, i.e. "mint" (add liquidity) and "burn" (remove liquidity).
-       - amounts: A list where each item is the amount added or removed of the pool token at the same index (from the "pool_tokens" list).
+       - amounts: A mapping between the pool token identifier and the amount added or removed.
```

Comment:
If we do go with zero amount not shown in the mapping it needs to be documented here.